doc/ref/states/requisites.rst

```diff
@@ -723,7 +723,7 @@ be installed. Thus allowing for a requisite to be defined "after the fact".
 
 .. code-block:: sls
 
-    {% for cfile in salt.pillar.get('nginx:config_files') %}
+    {% for cfile in salt['pillar.get']('nginx:config_files') %}
```

Comment:
`black` rules in favour of double quotes. Documentation should ideally be consistent with that.

```py
salt["pillar.get"]("nginx:config_files")
```

There are other examples in this file and PR.

Reply:
Oh, well nevermind then.